.circleci/pytest.ini

```diff
@@ -1,3 +1,3 @@
 [pytest]
-addopts = -s -v --junitxml=./reports/pytest-results.xml --strict-markers --durations=0 --cov=nucypher --cov-report xml:reports/coverage.xml
+addopts = -s -v --junitxml=./reports/pytest-results.xml --strict-markers --durations=0 --cov=nucypher --cov-report xml:reports/coverage.xml --timeout 580
```

Comment:
> `--timeout 580`

That's... very specific

Reply:
Heh - I chose this number because it's 20 seconds less than the forced timeout on circleCI.

Reply:
Why do we need one more timeout then?

Reply:
If we can catch the timeout with pytest-timout - it will cause the test to fail via `pytest.fail`, and dump the stack for inspection.   When circleCI issues a timeout, it's halts the docker container running the job leaving no clues as to why the timeout happened.  This is ultimately how I was able to discover the cause of issue #2316

Reply:
Ah, I see, thanks for the explanation.